client/src/pages/home/components/events/constants.jsx

```diff
@@ -11,16 +10,8 @@ import summerHiring from 'assets/home-page/events/summer_hiring.jpeg';
 
 export const UPCOMINGEVENTS = {
 	title: 'Upcoming Events',
-	events: [
-		{
-			title: 'Summer Hiring',
```

Comment:
Just remove the squiggly brackets and makes `events` an empty array. It is not right to change data type from array `[ ]` to string `' '`